src/reset.css

```
@@ -0,0 +1,4 @@
+body,
+div {
```

div doesn't have default margins
```suggestion
body {
```